assignments/P0.md

```diff
@@ -24,7 +24,7 @@ The objective of this assignment is to build a UNIX style shell.
 
     * You may find it prudent, and later expedient, to implement tests to ensure correctness of the code at each level
 
-    * Use `git commit --amend` to fix mistakes in the most recent commit, and explore `man git-rebase` for more complex revision
+    * Use `$ git commit --amend` to fix mistakes in the most recent commit, and explore `$ man git-rebase` for more complex revision
```

Comment:
This is not on its own line, as are most of the other changes to p0 and p1. Did you forget to add those files before committing again?